variables.tf

```diff
@@ -3,10 +3,22 @@
 ##############################################################################
 
 variable "name" {
-  description = "Name for VPC"
+  description = "Name for VPC."
   type        = string
 }
 
+variable "create_vpc" {
```

Comment:
I don't think we need `create_vpc` and `existing_vpc_id `. Just do if `existing_vpc_id ` is null create a new vpc. @ocofaigh do you agree?

Reply:
We do need it - we will face issues where if you install the VPC in same terraform apply, this will fail saying it can't compute the plan on an unknown value. so hence it needs a boolean (we found this out the hard way before)